src/globals/scss/_theme-tokens.scss

```diff
@@ -173,6 +173,8 @@
   $data-table-heading-transform: uppercase !default !global;
   $data-table-heading-border-bottom: 1px solid $brand-01 !default !global;
   $data-table-row-height: 2rem !default !global;
+  $data-table-zebra-color: #fcfcfc !default !global;
```

Comment:
Is there a token for the zebra color we could use? Or IDL color variable?

Reply:
There isn't a token

Reply:
We should flag this if a color is being used without a token, or without a variable from IDL.

Reply:
I asked about this before and this is what Tyler said: _Yes, the zebra should be the `#fcfcfc` it’s not a color associated with a token. It follows the the same contrast ratio as the v9 zebra stripe colors thought. Any existing token colors did not work well and messed with hover and selected states._

Reply:
@jnm2377 thanks for the info! My assumption is that this is an interaction/hover color coming from IDL, is that right @designertyler? Do you know which one it's for? Maybe a gray10 hover?

Reply:
Having a hard time finding zebra in the sketch file, is there a specific artboard where it's located?

Reply:
@joshblack The zebra stripes are in the "Process" page in the sketch file. The `fcfcfc` color for the zebra stripe doesn't exist in the IDL palette. We tried using gray10 hover, gray20, and white for the alternating colors, but nothing in the palette gave the subtle contrast the v9 zebra data table had. I suggested `fcfcfc` as a new color to match the same contrast ratio the zebra stripes did in v9.

Reply:
Gotcha, as long as brand is cool with it then 👍